layers.rmd

```diff
@@ -481,7 +481,7 @@ ggplot(diamonds, aes(price, colour = cut)) +
   theme(legend.position = "none")
 ```
 
-The result of this plot is rather surprising: low quality diamonds seem to be more expensive on average. We'll come back to this result in Section \@ref(trend).
+The result of this plot is rather surprising: low quality diamonds seem to be more expensive on average.
```

Comment:
If this example no longer reappears later, it might be worth adding a sentence here explaining the counter-intuitive result?

Reply:
Or maybe a footnote? Want to do that in a new PR?